engine/src/hopeit/dataobjects/__init__.py

```diff
@@ -1,6 +1,6 @@
 """
 Data Objects type abstractions.
-A Dataobject is basically any object wrapped in a Dataclass and that is able
+A Dataobject is basically any object wrapped in a Pydantic BaseModel and that is able
```

Comment:
A Dataobject is basically any object like a regular dataclass but adapted to
implement `pydantic.BaseModel` functionallity plus specific `hopeit.engine` functionallity
to make it able to be stored...